Libraries/Network/RCTNetwork.xcodeproj/project.pbxproj

```diff
@@ -226,6 +227,7 @@
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
 					"$(SRCROOT)/../../React/**",
+					"$(SRCROOT)/../Image/**",
```

Comment:
Yeah, we need to add `Libraries/Image/` to the header search path so we can `#import "RCTImageLoader.h"`